django_thumbor/templatetags/thumbor_tags.py

```diff
@@ -27,3 +27,12 @@ def thumbor_url(image_url, **kwargs):
         kwargs['filters'] = _parse_filters(filters)
 
     return generate_url(image_url=image_url, **kwargs)
+
+@register.assignment_tag()
+def assign_thumbor_url(image_url, **kwargs):
```

Comment:
Ah, sorry, you're right. I needed that for our use case. I didn't want to skip the discussion, though. Do you think it makes sense? I'll move it to a separate PR and add documentation...